terraform/modules/compute_plane/main.tf

```diff
@@ -80,13 +74,5 @@ resource "docker_container" "polling_agent" {
     }
   }
 
-  healthcheck {
-    test         = ["CMD", "bash", "-c", "exec 3<>\"/dev/tcp/localhost/1080\" && echo -en \"GET /liveness HTTP/1.1\r\nHost: localhost:1080\r\nConnection: close\r\n\r\n\">&3 && grep Healthy <&3 &>/dev/null || exit 1"]
-    interval     = "5s"
-    timeout      = "3s"
-    start_period = "20s"
-    retries      = 5
-  }
```

Comment:
Don't you want to keep it on Linux?

Reply:
I did not want to introduce new variable and container crashes and restart with the current implementation